rollup.config.js

```diff
@@ -29,20 +28,16 @@ export default [{
 		name: 'visualHeatmap'
 	}],
 	plugins: [
-		// babel({
-	    //   babelHelpers: 'bundled',
-	    //   exclude: 'node_modules/**', // Don't transpile node_modules
-	    // }),
-		// commonjs(),
 		typescript(),
-		eslint({
-			fix: true,
-			throwOnError: true
-		}),
+		// eslint({
```

Comment:
@nswamy14 Need to discuss on this since eslint needs to be configured to handle Typescript